test/suite/test_tiered03.py

```diff
@@ -42,6 +42,8 @@ class test_tiered03(wttest.WiredTigerTestCase, TieredConfigMixin):
     # sharing would probably need to be reworked.
     uri = 'file:test_tiered03'
 
+    storage_sources = gen_tiered_storage_sources(wttest.getrandom_prefix(), 'test_tiered03', tiered_only=True)
```

Comment:
I don't like passing the class name as an argument, but I do not have a better solution in my mind. In initial change i was trying to derive the calling function name using stack trace, but that is not going to work here.

Reply:
Yeah unfortunately that solution didn't work after the refactoring and moving the storage_sources list to the helper file.